runtime/help/plugins.md

```diff
@@ -62,6 +62,10 @@ that micro defines:
 * `onBufferOpen(buf)`: runs when a buffer is opened. The input contains
    the buffer object.
 
+* `onBufferOptionChanged(buf, option, old, new)`: runs when a option of the
+   settings of a buffer has changed. The input contains the buffer object,
```

Comment:
"a option of the settings of a buffer" sounds exotic (and not exactly grammatically correct). Just "runs when an option of the buffer has changed"?